src/main/kotlin/uk/gov/justice/digital/hmpps/nomisuserrolesapi/config/NomisUserRolesApiExceptionHandler.kt

```diff
@@ -219,6 +222,36 @@ class NomisUserRolesApiExceptionHandler {
       )
   }
 
+  @ExceptionHandler(ReusedPasswordException::class)
+  fun reusedPasswordException(e: ReusedPasswordException): ResponseEntity<ErrorResponse> {
+    log.debug("Password reused error (400) returned", e)
```

Comment:
it's really rubbish of app insights but when logging at `debug` like this with `e` it appears in `exceptions` unnecessarily.  We went through a task a while ago of changing all the exception handler stuff to therefore log `e.message` instead to avoid it